src/main/resources/db/migration/V5_6__delete_message_wrong_person.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM DIALOGMELDINGOPPLYSNINGER WHERE id='b3d262ac-a5c2-41e7-bd81-bcaecae63842';
```

Comment:
Er IDen en uuid i denne dben?

Reply:
Ja, ble gjort annerledes her "back in the days".